api/lib/opentelemetry/baggage/noop_builder.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Baggage
+    # No op implementation of Baggage::Builder
+    class NoopBuilder
```

Comment:
Does the `NoopBuilder` also require an implementation of `entries`?
```suggestion
    class NoopBuilder
      def entries; {}; end

```

Reply:
🤔 or should `entries` be `@api private` in `Builder`? I think it is only meant to be called from `Manager`.

Reply:
It shouldn't need one. An operational manager will call Builder#entries in order create a new context containing them, but the NoopManager does nothing. I could add an @api private to the existing `entries` on the operational Manager to clear that up as it's not meant to be exposed to the user.